src/main/java/internet/shop/dao/OrderDao.java

```diff
@@ -0,0 +1,19 @@
+package internet.shop.dao;
+
+import internet.shop.model.Order;
+import internet.shop.model.User;
+import java.util.List;
+import java.util.Optional;
+
+public interface OrderDao {
+
+    Order completeOrder(Order order);
```

Comment:
```suggestion
    Order create(Order order);
```
так як тут не повинна міститись якась додаткова логіка а ти просто додаєш в базу ордер який тобі приходить